config/manager/persistent-volume.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
We may not need to create PV for internally usage https://github.intuit.com/pages/kubernetes/IKS/iks_storage_options/